R/read_csv.R

```diff
@@ -204,41 +206,44 @@ read_cmdstan_csv <- function(files,
       all_draws <- 1
     }
 
+    vroom_args <- list(
```

Comment:
In order to not have to do try() multiple times, make list of vroom arguments so we only need one call to vroom() in the code. previously we had separate calls depending on generated quantities or not.